### PR TITLE
Add TAXPASTA_STANDARDISE support

### PR DESCRIPTION
**Pull request by jfy133**
So taxpasta merge doesn't fail when only a single sample is passed through the pipeline.

resolves https://github.com/nf-core/taxprofiler/issues/282

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/standardisation_profiles.nf [hunk 1]
```diff
@@ -35,7 +37,17 @@ workflow STANDARDISATION_PROFILES {
 
     ch_taxpasta_tax_dir = params.taxpasta_taxonomy_dir ? Channel.fromPath(params.taxpasta_taxonomy_dir, checkIfExists: true).collect() : []
 
-    TAXPASTA_MERGE (ch_input_for_taxpasta, ch_taxpasta_tax_dir, [])
+    ch_input_for_taxpasta = ch_prepare_for_taxpasta
+                        .dump(tag: "pre-branch")
```
Comment (Midnighter) [inline subworkflows/local/standardisation_profiles.nf:41]: We were removing `dump` calls, right?

**Review by Midnighter — APPROVED** [review]
Super solid work, thanks 😃

File: subworkflows/local/standardisation_profiles.nf (hunk 1 above)
Comment (jfy133) [reply subworkflows/local/standardisation_profiles.nf:41]: Oops yes!

Comment (jfy133) [inline subworkflows/local/standardisation_profiles.nf:41]:
```suggestion
```

**Pull request merged**